website/src/utils/Intl.ts

```diff
@@ -0,0 +1,5 @@
+const IntlFormatter = new Intl.NumberFormat('en-GB', { maximumFractionDigits: 2 });
```

Comment:
Why should it be exclusively in en-GB? Why not en-US or let the browser decide based on the user's locale?

Reply:
How can I do that? Although ideally it should stick to en-US or en-GB or even better, the locale of the website if we'll add i18n

Reply:
You use `undefined` for automatic detection, but honestly, you can also use `en-US`, so the formatting is consistent with the rest of the website.